### ISSUE TITLE
Stop returning REST response with `Has Many`, `Belongs To`, or `Many to Many` by default, because they can be controlled if query params `bt`, `hm`, or `mm` are set

### ISSUE THREAD

**Issue by TechMasterElvy** [nocodb/nocodb#856]
please provide the following details :
NocoDB used as docker : false (heroku)
NocoDB version : 0.83.5

Database used in NC_DB URL  pg
Project was created  by clicking : New Project by connecting to external database
Database on which spreadsheet is created : pg
OS on which NocoDB is running : heroku hobby dynamo
Node.js version if running as node : v12.22.6
Database version : 
* NoCoDB: PostgreSQL 13.5 (Ubuntu 13.5-2.pgdg20.04+1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0, 64-bit
* Project DB: PostgreSQL 14.1 on aarch64-unknown-linux-gnu, compiled by gcc (Ubuntu 10.3.0-1ubuntu1~20.04) 10.3.0, 64-bit

**Steps To Reproduce**
1. Query GET for a resource that has bt, hm or mm without setting bt, hm, mm `api/v1/countries` which `has many`.`places`

**Expected behavior**
JSON should return the entity without any joins

**Actual behaviour**
All relations are returned with no apparent way to turn it off
```[
  {
    "Id": 1,
    "title": "Country A",
    "CreatedAt": "2021-12-09T11:05:44.540Z",
    "UpdatedAt": "2021-12-12T11:00:23.904Z",
    "is_published": true,
    "placesList": [
      {
        "Id": 4,
        "title": "Place A",
        "CountriesId": 1
      },
      {
        "Id": 6,
        "title": "Place B",
        "CountriesId": 1
      },
      {
        "Id": 7,
        "title": "Place C",
        "CountriesId": 1
      }
    ]
  },
  {
    "Id": 2,
    "title": "Country B",
    "CreatedAt": "2021-12-09T11:05:50.562Z",
    "UpdatedAt": "2021-12-12T11:00:39.131Z",
    "is_published": true,
    "placesList": []
  }
]
```

I tried setting bt, hm, mm to `undefined` or `null` or `false` to disable this behaviour but go this error when I try

```{
  "msg": "Cannot read property 'columnToAlias' of undefined"
}
```

Node: **v12.22.6**
Arch: **x64**
Platform: **linux**
Docker: **false**
Database: **pg**
ProjectOnRootDB: **false**
RootDB: **pg**
PackageVersion: **0.83.5**

**Comment by pranavxc** [comment]
You can pass `hm` as empty string to ignore has many list

**Comment by TechMasterElvy** [comment]
> You can pass `hm` as empty string to ignore has many list
@pranavxc Just tried calling the following
* `/api/v1/countries?hm=%20` (single space) 
* `api/v1/countries?hm=%22%22` (double quotes "")
* `/api/v1/countries?hm='\'''\''' \` (single quotes '')

 but still got the same error 
```{
  "msg": "Cannot read property 'columnToAlias' of undefined"
}
```

**Comment by pranavxc** [comment]
Just pass empty string means not even whitespace

```
/api/v1/countries?hm=&limit=25
/api/v1/countries?hm=
```

**Comment by pranavxc** [comment]
Fixed in `0.83.6`
